addons/popochiu/migration/migration/popochiu_migration.gd

```diff
@@ -32,6 +32,10 @@ func _init() -> void:
 #endregion
 
 #region Virtual ####################################################################################
+func _is_migration_needed() -> bool:
+	return true
```

Comment:
Uhmmm... I think having a function which only returns true is redundant.

If I guess why you are doing this (like, forcing a migration for this very version), you may just use a constant at line 68 (see comment below).

Reply:
This is a virtual function. The idea is that migrations which inherit from this class can override it to add additional logic to verify if they are or not needed. Loof for example the implementation of `popochiu_migration_4.gd`.

Reply:
ok, ignore me! I understand now!